### PR TITLE
chore: add test that `chromeWebSecurity` affects CORS requests

### PR DESCRIPTION
**Pull request by flotwig**
- Test for #8399

### User facing changelog

N/A

### Additional details

Fixed by #8406 but needed to add a test

Decaffed web_security_spec

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has the original issue been tagged with a release in ZenHub? 
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Review by Bkucera — CHANGES_REQUESTED** [review]
could we just use `127.0.0.2` instead of `foo.com`? since this is still making an OPTIONS request right?

File: packages/server/test/e2e/6_web_security_spec.js [hunk 1]
```diff
@@ -38,72 +37,66 @@ const onServer = function (app) {
 </html>\
 `)
   })
+
+  app.get('/cors', (req, res) => {
+    res.send(`<script>
+      fetch('https://www.foo.com:44665/cross_origin')
```
Comment (flotwig) [inline packages/server/test/e2e/6_web_security_spec.js:43]:
```suggestion
      fetch('https://127.0.0.2:44665/cross_origin')
```

**Comment by flotwig** [comment]
should work too, updated

**Comment by Bkucera** [comment]
not sure why the percy snapshot difference is happening

**Review by Bkucera — APPROVED** [review]

**Pull request merged**